sdk/src/test/tests/TestManagedNativeClass.ls

```diff
@@ -85,69 +83,137 @@ final native class MyChildManagedNativeClass extends MyManagedNativeClass {
     {
         super.intProperty = value + 2;
     }
-
-    
 }
 
-class TestManagedNativeClass extends LegacyTest
+class TestManagedNativeClass
```

Comment:
The new tests usually have `Test` at the end of the name, but it's not a big deal.